rollup.umd.config.js

```diff
@@ -0,0 +1,23 @@
+import { uglify } from 'rollup-plugin-uglify';
+
+const umdConfig = {
```

Comment:
Maybe not just umd output? ESM and cjs are all necessary imo.

Reply:
```
"build:cjs": "rollup -i src/index.js -o lib/index.js -f cjs",
"build:cjs-polyfill": "rollup -i src/polyfill.js -o lib/polyfill.js -f cjs",
```

cjs output is already supported, will probably add esm output support in a separate pr.